scripts/cibuild.sh

```diff
@@ -18,23 +18,31 @@ SYNTAX=$(find . -name "*.py" -type f -exec python -m py_compile {} \; 2>&1)
 if [[ ! -z $SYNTAX ]]; then
 	echo -e "$SYNTAX"
 	echo -e "\n$(info) detected one or more syntax errors; failing build."
-	EXIT=1
+	exit 1
 fi
 
 # Check all built-in python source files for PEP 8 compliance
 flake8 development/ nautobot/ tasks.py
 RC=$?
 if [[ $RC != 0 ]]; then
 	echo -e "\n$(info) one or more PEP 8 errors detected; failing build."
-	EXIT=$RC
+	exit $RC
 fi
 
 # Check that all files conform to Black.
 black --check development/ nautobot/ tasks.py
 RC=$?
 if [[ $RC != 0 ]]; then
 	echo -e "\n$(info) one or more Black errors detected; failing build."
-	EXIT=$RC
+	exit $RC
+fi
+
+# Dockerfile lint with hadolint
+docker run --rm -i ghcr.io/hadolint/hadolint < docker/Dockerfile
```

Comment:
My concern with adding this here tightly couples Docker to our development environments. If there is a way to install `hadolint` for use outside of a container as part of the development requirements, I'd prefer that.